back/src/companies/queries/userRole.ts

```diff
@@ -0,0 +1,20 @@
+import { prisma } from "../../generated/prisma-client";
+
+/**
+ * Returns the role (ADMIN or MEMBER) of a user
+ * in a company.
+ * Returns null if the user is not a member of the company.
+ * There should be only one association between a user
```

Comment:
Remarque annexe: malheureusement ce n'etait pas toujours le cas avant... :( Faudra qu'on s'assure que ca ne peut plus arriver et qu'on fasse un script d'upgrade pour faire le ménage et laisser pour chaque user l'association donnant le droit "maximal" (dans une future PR of course)

Reply:
Yes bien vu, j'ai ajouté une carte https://trello.com/c/rM9PT5aT